proxyretry_test.go

```diff
@@ -154,7 +155,17 @@ func TestQueryWithRetrySuccess(t *testing.T) {
 	if err != nil {
 		t.Errorf("The execution with retry failed, %v", err)
 	}
-	assert.Equal(t, srw.statusCode, 200)
+	assert.Equal(t, 200, srw.statusCode)
+	assert.Equal(t, 1, int(s.host.counter.load()))
+	assert.Equal(t, 0, int(s.host.penalty))
+	// should be counter + penalty
+	assert.Equal(t, 1, int(s.host.load()))
+
+	assert.Equal(t, 0, int(erroredHost.counter.load()))
+	assert.Equal(t, 5, int(erroredHost.penalty))
```

Comment:
You should use the **penaltySize** variable instead of 5, it would make the test more readable

Reply:
done